homepage/homepage.go

```diff
@@ -99,14 +101,26 @@ func init() {
 	}
 
 	// Unemployment
-	mainFigureMap["MGSX"] = MainFigure{
+	mainFigureMap["unemp"] = MainFigure{
 		uris:               []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/mgsx/lms"},
 		trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/fuu8/lms",
 		datePeriod:         mapper.PeriodMonth,
 		data:               zebedee.TimeseriesMainFigure{},
 		differenceInterval: mapper.PeriodYear,
 	}
 
+	if cfg.EnableUpdatedMainFigures {
+		mainFigureMapCopy := mainFigureMap["emp"]
+		mainFigureMapCopy.uris = []string{"/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2pw/lms"}
+		mainFigureMapCopy.trendURI = "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qm/lms"
+		mainFigureMap["emp"] = mainFigureMapCopy
+
+		mainFigureMapCopy = mainFigureMap["unemp"]
+		mainFigureMapCopy.uris = []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/s2pu/lms"}
+		mainFigureMapCopy.trendURI = "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qk/lms"
+		mainFigureMap["unemp"] = mainFigureMapCopy
+	}
+
```

Comment:
I would have preferred this in a distinct `if/else` block...

**TL;DR**
- It would be easier to read 
- Easier to remove the old employment stats later on
- Less reliant on being on a certain order i.e. you need lines 96-98 & 107-109 set first 
- Would save on rewriting to the same variable i.e. instantiated on line 113 and reused on 118. 

```go
if cfg.EnableUpdatedMainFigures {
		// Employment
		mainFigureMap["emp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2pw/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qm/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}

		// Unemployment
		mainFigureMap["unemp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/s2pu/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/s2qk/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}
	} else {
		// Employment
		mainFigureMap["emp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/lf24/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/fux7/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}

		// Unemployment
		mainFigureMap["unemp"] = MainFigure{
			uris:               []string{"/employmentandlabourmarket/peoplenotinwork/unemployment/timeseries/mgsx/lms"},
			trendURI:           "/employmentandlabourmarket/peopleinwork/employmentandemployeetypes/timeseries/fuu8/lms",
			datePeriod:         mapper.PeriodMonth,
			data:               zebedee.TimeseriesMainFigure{},
			differenceInterval: mapper.PeriodYear,
		}
	} 
```